cli/src/main.js

```diff
@@ -29,42 +28,48 @@ const commands = {
   schema: schemaCommand,
 };
 
-function parseArgs() {
-  const parser = new argparse.ArgumentParser();
-  const subparsers = parser.addSubparsers({
-    title: 'commands',
-    dest: 'command_name',
-  });
+const programName = path.basename(process.argv[1]);
```

Comment:
Is `argv[1]` correct?  I imagine this will be different if you run it via `node <script> <subcommand>` vs `hz <subcommand>`.